e3nn/util/codegen/_mixin.py

```diff
@@ -96,6 +107,8 @@ def __setstate__(self, d):
                 # Load the TorchScript IR buffer
                 buffer = io.BytesIO(buffer)
                 smod = torch.jit.load(buffer)
+                if isinstance(smod, fx.GraphModule):
+                    smod = torch.jit.script(jitable(smod))
```

Comment:
since we're in `__setstate__` it can't be a GraphMod, it has to be a scriptmodule... since thats the only thing we can save

Reply:
```suggestion
```